packages/block-editor/src/components/block-switcher/index.js

```diff
@@ -177,7 +177,7 @@ export class BlockSwitcher extends Component {
 								{ possibleBlockTransformations.length !== 0 && (
 									<PanelBody
 										title={ __( 'Transform To:' ) }
-										initialOpen
+										alwaysOpen={ ! hasBlockStyles }
```

Comment:
Thanks for the PR

I believe this can be replaced with the `opened` prop without introducing any new prop.

Reply:
I tried that at first, but in the Inserter component, `opened` is used to control open state from outside, with panels receiving `opened={true}` or `opened={false}` depending on user interaction -- e.g.:

`https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/inserter/menu.js#L125-L133`

This update removes the toggle button in the panel heading when `opened` is set true, which means that once opened, a thee panel can no longer be closed.

Reply:
Why use a PanelBody in this case at all, if we want to disable the toggling. In that case, we can just use a Card component. 

Noting that this uses a panel because that menu can show style variations too (check the button block for instance), also let's get some design thoughts on how we should move forward..

Reply:
This accounts for the style variations by setting `alwaysOpen` to true if block styles don't exist. 

This accounts for the style variations by setting `alwaysOpen` to true if block styles don't exist. 

If the "Transform To" PanelBody is being refactored out as @jasmussen mentioned, and there are alternatives that do what an `alwaysOpen` panel body would do, then this change might not be needed. I think we'd still want to update the Panel README's `opened` description, though, because "the panel will be prevented from being closed" gives the impression that we are intentionally providing an unclosable panel body (which shouldn't have the toggle button). Maybe the README could point to an alternative component for this need.